test/DropdownButtonSpec.jsx

```diff
@@ -12,7 +12,7 @@ describe('DropdownButton', function () {
 
   afterEach(function() {
     if (instance && ReactTestUtils.isCompositeComponent(instance) && instance.isMounted()) {
-      React.unmountComponentAtNode(instance.getDOMNode().parent);
+      React.unmountComponentAtNode(instance.getDOMNode());
```

Comment:
`getDOMNode` isn't depreciated in createClass, as far as I know, so there is no warning. I can be changed but it isn't necessary